baseapp/baseapp.go

```diff
@@ -847,12 +847,15 @@ func createEvents(msg sdk.Msg) sdk.Events {
 		msgEvent = msgEvent.AppendAttributes(sdk.NewAttribute(sdk.AttributeKeySender, msg.GetSigners()[0].String()))
 	}
 
-	// here we assume that routes module name is the second element of the route
-	// e.g. "cosmos.bank.v1beta1.MsgSend" => "bank"
-	moduleName := strings.Split(eventMsgName, ".")
-	if len(moduleName) > 1 {
-		msgEvent = msgEvent.AppendAttributes(sdk.NewAttribute(sdk.AttributeKeyModule, moduleName[1]))
+	// verify that events have no module attribute set
+	if _, found := events.GetAttributes(sdk.AttributeKeyModule); !found {
```

Comment:
Does all tx event of ibc include `module` attribute?

Reply:
The supported version of `Finschia/ibc-go/v4` is `v4.3.1`. Hence, I will check it on that version.

By the way, this PR would not remove any information included before.

Reply:
> Does all tx event of ibc include `module` attribute?

No, the current version, `v4.3.1`, does not include `module` attribute in all its events. Some logic lacks the event.

Reply:
If then, does the all ibc events display no ibc module name?

Reply:
Some logics emit the events, but others don't.

Reply:
I will add logic not to append attribute `module` in case of `ibc`. It would be addressed by another PR.